packages/design-system/package.json

```diff
@@ -14,12 +14,14 @@
   },
   "scripts": {
     "build": "npm run build:theme",
-    "build:vue": "vue-cli-service build --target lib ./src/main.js --report",
+    "build:vue": "vite build",
```

Comment:
can you please pull master and lmk? there's a missing dep here fixed in master that would allow me to deploy this